CONTRIBUTING.md

```diff
@@ -124,6 +124,9 @@ TODO
 If the ebuild you are referencing is not in either of those locations, please
 file a PR into ROS-Overlay to add it and any needed dependencies to the tree.
 
+Note that `pip` cannot be used to install system packages on Gentoo.
+If the package is not present in the Gentoo package repository, please create an issue on the ros/ros-overlay repository so that it may be created at a later date, omit the key from the new rosdep rule, and link the created issue in the PR which is adding the rule.
```

Comment:
```suggestion
If the package is not present in the Gentoo package repository, please create an issue on the https://github.com/ros/ros-overlay repository so that it may be created at a later date, omit the key from the new rosdep rule, and link the created issue in the PR which is adding the rule.
```